packages/main/src/RatingIndicator.ts

```diff
@@ -187,6 +187,18 @@ class RatingIndicator extends UI5Element {
 	@property({ type: Boolean })
 	required!: boolean;
 
+	/**
+	 * Defines the tooltip of the component.
+	 *
+	 * @type {string}
+	 * @name sap.ui.webc.main.RatingIndicator.prototype.tooltip
+	 * @defaultvalue ""
+	 * @public
+	 * @since 1.19.0
+	 */
+	@property({ defaultValue: "" })
```

Comment:
you can skip the default value here in the decorator

Reply:
done